### PR TITLE
Add some options for bundle when create the installer on windows

### PR DESCRIPTION
**Pull request by MikeLing**

### CODE REVIEW

File: gui/build.py [hunk 1]
```diff
@@ -80,14 +80,21 @@ def do_bundle(options):
         if os.path.isdir(dirname):
             shutil.rmtree(dirname)
     # create a intaller
+    if IS_WIN:
```
Comment (parkouss) [inline gui/build.py]: We don't need the IS_WIN check yet - for now this part of the script is windows only (we will make it work for linux with the next bug). So please remove it for now. Also, the default values for python_path and nsis_path can be defined as a default= argument in the add_argument call, so you can just remove all this block.

File: gui/build.py [hunk 2]
```diff
@@ -124,6 +131,10 @@ def parse_args():
 
     bundle = subparsers.add_parser('bundle',
                                    help='bundle the application (freeze)')
+    bundle.add_argument('--python-path', dest='python_path',
+                        help='your python path on system')
+    bundle.add_argument('--nsis-path', dest='nsis_path',
+                        help='your nsis path on system')
```
Comment (parkouss) [inline gui/build.py]: please put the default values here, it will be simpler.

File: gui/build.py [hunk 3]
```diff
@@ -80,14 +80,21 @@ def do_bundle(options):
...
+        if options.python_path:
+            python_path = options.python_path
+        if options.nsis_path:
+            nsis_path = options.nsis_pat
     python_lib_path = os.path.join(
-        os.getenv('MOZREGUI_PYTHONPATH', "C:\\Python27"),
+        os.getenv('MOZREGUI_PYTHONPATH', python_path),
         "Lib"
-    )
+        )
```
Comment (parkouss) [inline gui/build.py]: you can let the parenthesis where they were, it was fine.

File: gui/build.py [hunk 4]
```diff
@@ -80,14 +80,21 @@ def do_bundle(options):
...
+    if IS_WIN:
+        python_path = 'C:\\Python27'
+        nsis_path = 'C:\\NSIS'
+        if options.python_path:
+            python_path = options.python_path
+        if options.nsis_path:
+            nsis_path = options.nsis_pat
     python_lib_path = os.path.join(
-        os.getenv('MOZREGUI_PYTHONPATH', "C:\\Python27"),
+        os.getenv('MOZREGUI_PYTHONPATH', python_path),
```
Comment (parkouss) [inline gui/build.py]: No need for the os.getenv now - we are replacing this with options.nsis_path.

File: gui/build.py [hunk 5]
```diff
@@ -80,14 +80,21 @@ def do_bundle(options):
...
         "Lib"
-    )
+        )
     makensis_path = os.path.join(
-        os.getenv('MOZREGUI_NSISPATH', "C:\\NSIS"),
+        os.getenv('MOZREGUI_NSISPATH', nsis_path),
         "makensis.exe"
-    )
+        )
```
Comment (parkouss) [inline gui/build.py]: same thing here - let the parenthesis where they were, and please remove the os.getenv call.

File: gui/build.py (hunk 1 above)
Comment (MikeLing) [inline gui/build.py]: Hm, I see. But when we call bundle under linux, the default argument will be also given to bundle if we forget to give a python path. So, why don't we set different default value of path when we call different bundle funtion?

Comment (parkouss) [inline gui/build.py]: Well, we will! But it is for the next patch - for now, the bundle command is only used in windows, so let keep things simple.

Comment (MikeLing) [inline gui/build.py]: I mean you can treat this patch as a windows bundle function.

> python_path = 'C:\\Python27'
> nsis_path = 'C:\\NSIS' 
And, we when bundle be called under Linux system.

> python_path = /usr/lib/python2.7

Comment (parkouss) [inline gui/build.py]: The goal of this patch is to remove the environment variables used to use command line arguments instead. Let's just do this for now, and not something else - it is easy to do, and easy to review. Let's talk on irc about that!

Comment (MikeLing) [inline gui/build.py]: I see. :)

Sorry, I have some classes this afternoon. I will send you email after that. Let's fix this issue first. :D

**Pull request merged**